backend/src/main/java/saci/domain/model/Level.java

```diff
@@ -0,0 +1,42 @@
+package saci.domain.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@Entity
+@Table(name = "levels")
+@NoArgsConstructor
+@AllArgsConstructor
+public class Level {
+    @Id
```

Comment:
Domain classes should not contain database detail implementations (@vascolusitan can explain this to u)